README.md

```diff
@@ -1,5 +1,8 @@
 # Insights Agent Desktop App
 
+The Build justly Project and Georgia Hispanic Chamber of Commerce, partnered to launch and execute a technology study. The study will endeavor to not only to better understand the barriers/frictions and interests/needs to technology of Hispanic small business owners in the state of Georgia but will also work to develop specific ways to bring down the barriers to technology competency.  This study, will be measuring efforts through both opt-in telemetry collection alongside on-going qualitative interviews and surveys which will be shared publicly. 
```

Comment:
I would just keep this focused on buildJUSTLY since we are hoping this will be expanded to other partners in the future.